src/richText.js

```diff
@@ -145,8 +113,17 @@ define([
      * if the editor does not exist.
      *
      * @param input - editor jQuery HTML element.
-     * @param form - form object; used for creating bubbles.
-     * @param options - rich text bubble options.
+     * @param form - form object, functions used:
```

Comment:
Agree, this either needs a different name, or it should be a real form instance. It seems like a recipe for confusion/bugs to be passing around things that look like forms but have different methods.